src/Controls/src/Build.Tasks/Controls.Build.Tasks.csproj

```diff
@@ -19,7 +19,7 @@
 
 	<ItemGroup>
 		<PackageReference Include="Mono.Cecil" Version="0.11.5" PrivateAssets="all" GeneratePathProperty="true" />
-		<PackageReference Include="System.CodeDom" Version="7.0.0" PrivateAssets="all" GeneratePathProperty="true" />
+		<PackageReference Include="System.CodeDom" Version="8.0.0-rc.2.23479.6" PrivateAssets="all" GeneratePathProperty="true" />
```

Comment:
@mattleibow @StephaneDelcroix  I s it fine to update this or we need to sync with VS on this?